### PR TITLE
Arilacanienta fa22 coordinators update

### PR DESCRIPTION
**Pull request by AriLacanienta**
Added bios and profile pictures for new club coordinators Ari & Owen. Created a placeholder entry fro Anna.

/schedule 2021-10-18

### CODE REVIEW

File: club/coordinators.md [hunk 1]
```diff
@@ -102,7 +116,7 @@ img.default {
   <h2 style="margin-bottom: 1rem">Coordinator</h2>
   <div class="row" style="margin-bottom: 1rem">
     <div class="col-lg-3">
-      <img src="/assets/img/club_photos/Ayushi_Sharma.JPG" alt="Picture of Kori South" class="rounded shadow coordinator">
+      <img src="/assets/img/club_photos/Ayushi_Sharma.JPG" alt="Picture of Ayushi Sharma" class="rounded shadow coordinator">
```
Comment (KVSRoyal) [inline club/coordinators.md:119]: Awesome! Definitely always make sure the alt text is updated.

File: club/coordinators.md [hunk 2]
```diff
@@ -128,15 +142,43 @@ img.default {
 </div>
 
 <div style="margin-bottom: 1rem">
-  <h2 style="margin-bottom: 1rem">President</h2>
+  <h2 style="margin-bottom: 1rem">Coordinator</h2>
   <div class="row" style="margin-bottom: 1rem">
     <div class="col-lg-3">
-      <img src="/assets/img/club_photos/Kori.jpeg" alt="Picture of Kori South" class="rounded shadow coordinator">
+      <img src="/assets/img/club_photos/Ari_Lacanienta.jpg" alt="Picture of Ari Lacanienta" class="rounded shadow coordinator"><!--TODO: Ari Pic -->
```
Comment (KVSRoyal) [inline club/coordinators.md:148]: This seems out of order. They should be in alphabetical order by last name.

**Review by KVSRoyal — COMMENTED** [review]
Have you tested this with Jekyll locally on your machine?

File: club/coordinators.md (hunk 2 above)
Comment (AriLacanienta) [reply club/coordinators.md:148]: Oh okay. I just thought it made sense to have the president at the top.

**Pull request merged**